apps/user-office-frontend/src/components/proposal/ProposalSummary.tsx

```diff
@@ -181,7 +184,7 @@ function ProposalReview({ confirm }: ProposalSummaryProps) {
               }
             )();
           }}
-          disabled={submitDisabled}
+          disabled={submitDisabled || !firstSubmit}

```

Comment:
Can't we just use the `submitDisabled` variable instead of introducing new one called `firstSubmit`? For example add `setSubmitDisabled` in the `useState` declaration of it and set it to true at the same place where you `setFirstSubmit`

Reply:
Also would be good maybe to have some e2e test that covers this for the future.

Reply:
@bashanlam I think you can completly remove this `|| !firstSubmit` part from the disabled condition and leave just `submitDisabled` as it was.